pages/about/AboutContextHandler.inc.php

```diff
@@ -51,6 +51,55 @@ function index($args, $request) {
 
 		$this->setupTemplate($request);
 		$templateMgr = TemplateManager::getManager($request);
+
+		$templateMgr->assign('journal', $context);
+		$templateMgr->assign('description', $context->getLocalizedSetting('description'));
+		$templateMgr->assign('aboutJournal', $context->getLocalizedSetting('aboutJournal'));
```

Comment:
I used this technique for most pages, except the `contact.tpl`. There is a [lot of conditional logic](https://github.com/pkp/pkp-lib/pull/1448/files#diff-30d735e87ca2fed9b71ee334a8597bd7R37) in that template and I feel like the template gets hard to read when you need to make the long call every time.

I also pass a `$submissionsChecklist` to the `submissions.tpl` because that undergoes a bit of [handling](https://github.com/pkp/pkp-lib/pull/1448/files#diff-bf758b48a6ee697fd8589ac5dd08a23aR74) behind the scenes.